src/app/containers/NewStoryPromo/Promo/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { node } from 'prop-types';
+import Timestamp from '#components/Promo/timestamp';
+import SectionLabel from '#legacy/psammead-section-label/src';
+import Content from '#app/containers/StoryPromo/LinkContents';
+import LiveLabel from '#legacy/psammead-live-label/src';
+import H3 from '../PromoAtoms/Headings.styles';
+import {
+  ListItem,
+  OrderedList,
+  UnorderedList,
+} from '../PromoAtoms/List.styles';
+import Image from '../PromoAtoms/Image.styles';
+import Link from '../PromoAtoms/Link.styles';
+import MediaIndicator from '../PromoAtoms/MediaIndicator';
+import BoxWrapper from '../PromoAtoms/BoxWrapper.styles';
+
+const Promo = ({ children, Wrapper }) => {
+  return <Wrapper role="region">{children}</Wrapper>;
```

Comment:
Working on context now, but I have a small issue with styles that prevented me from fully using this.

Reply:
Doneish, I expanded it a bit. I believe this could be expanded even further once I figure out how to have a dynamic list component that allows us to customise the list for every Promo.